src/js/modules/activityModule.js

```diff
@@ -0,0 +1,107 @@
+import { createNoPoints, findNoPointsElement } from "../components/noPoints";
+import {
+  createScoreBlock,
+  findScoreBlockElement,
+  updateScore,
+} from "../components/scoreBlock";
+import nodeToArray from "../helpers/nodeToArray";
+
+const isTaskCompleted = (svgPath) => {
+  const completedImagePath = "sm1/notification_completed.svg";
+
+  return svgPath === completedImagePath;
+};
+
+const getItemScore = (name, regex) => {
+  const scoreText = name.replaceAll("\n", " ").match(regex)?.groups?.["score"];
+  return parseInt(scoreText) || undefined;
+};
+
+const getItemsScores = (tasks, regexForScoreAndPoints) => {
+  return tasks.map((task) => {
+    const taskChildNodes = nodeToArray(task.childNodes);
+
+    return taskChildNodes
+      .map((taskChildNode) => {
+        const svgPathOfItemChildNode =
+          taskChildNode.querySelector("svg").dataset.svgsPath;
+
+        if (isTaskCompleted(svgPathOfItemChildNode)) {
+          const itemScore = getItemScore(
+            taskChildNode.innerText,
+            regexForScoreAndPoints
+          );
+
+          return itemScore ?? 0;
+        } else {
+          return 0;
+        }
+      })
+      .reduce((accumulator, currentValue) => accumulator + currentValue, 0);
+  });
+};
+
+const postCounterToPage = (points, indexOfParent, parent) => {
+  const scoreBlockParent = parent[indexOfParent].querySelector("h2");
+
+  const scoreBlockElement = findScoreBlockElement(scoreBlockParent);
+
+  if (!scoreBlockElement) {
+    const scoreBlock = createScoreBlock(scoreBlockParent, points);
+    scoreBlockParent.append(scoreBlock);
+    return;
+  }
+
+  updateScore(scoreBlockParent, points);
+};
+
+const isTaskCorrect = (regexForScoreAndPoints) => {
+  const taskIcons = document.getElementsByClassName("avatar_event_icon");
+
+  nodeToArray(taskIcons).map((taskIcon) => {
+    const taskItem = taskIcon.parentElement.parentElement;
+
+    if (!isTaskCompleted(taskIcon.querySelector("svg").dataset.svgsPath)) {
+      return;
+    }
+
+    const taskTime = taskItem.querySelector("span.activity_time");
+
+    const taskName = taskItem.querySelector(".task_content").textContent;
+
+    const taskText = taskItem.querySelector(".text");
```

Comment:
```suggestion
    const taskTime = taskItem.querySelector("span.activity_time");
    const taskName = taskItem.querySelector(".task_content").textContent;
    const taskText = taskItem.querySelector(".text");
```
They are related so group them together